config/initializers/google_maps.rb

```diff
@@ -0,0 +1,2 @@
+# Set Google Maps API key for development
+ENV['GOOGLE_MAPS_API_KEY'] ||= 'AIzaSyAPJUuLWQVCWoTIB2UkPAfgQSmPkDho9H0'
```

Comment:
Oh, totally overlooked the app is using Foreman. I'll add it there, but would also keep the initializer (e.g. for test env).